Cargo.toml

```diff
@@ -25,14 +25,22 @@ cfg-if = "1.0.0"
 libc = "0.2.156"
 psm = { path = "psm", version = "0.1.7" }
 
-[target.'cfg(windows)'.dependencies.windows-sys]
+[target.'cfg(all(windows, not(target_arch = "arm64ec")))'.dependencies.windows-sys]
 version = ">=0.52.0, <0.60.0"
 features = [
     "Win32_System_Memory",
     "Win32_System_Threading",
     "Win32_Foundation",
 ]
 
+[target.arm64ec-pc-windows-msvc.dependencies.windows-sys]
+version = ">=0.59.0, <0.60.0"
+features = [
+    "Win32_System_Memory",
+    "Win32_System_Threading",
+    "Win32_Foundation",
+]
+
```

Comment:
Why not bump `windows-sys` dep accordingly for all windows targets?

Reply:
I'm fine with doing that, although I think that will be a semver breaking change. Let me know if that's what you prefer.